src/deploy/AWSDeployer.js

```diff
@@ -218,9 +218,8 @@ export default class AWSDeployer extends BaseDeployer {
     this.log.info(chalk`{green ok:} deleted deploy package {blueBright s3://${this._bucket}/${this._key}}.`);
   }
 
-  async createLambda() {
+  get functionConfig() {
```

Comment:
I don't like getters that contain too much logic, I would prefer an explicit:

```suggestion
  getFunctionConfig() {
```